app/routers/event.py

```diff
@@ -40,8 +40,11 @@ async def create_new_event(request: Request, session=Depends(get_db)):
     end = datetime.strptime(data['end_date'] + ' ' + data['end_time'],
                             '%Y-%m-%d %H:%M')
     user = session.query(User).filter_by(id=1).first()
-    user = user if user else create_user(username="u", password="p",
-                                         email="e@mail.com", language="",
+    user = user if user else create_user(username="u",
+                                         password="p",
+                                         email="e@mail.com",
+                                         language="",
+                                         lanauge_id=1,
```

Comment:
There is an error here - lanauge_id > language_id. @yammesicka shouldn't this have failed the checker?

Reply:
@OdeChan, it seems it didn't fail as (I think) the placeholder code there isn't run in a test, as one line before it gets the User object from `user = session.query(User).filter_by(id=1).first()`, which seems to be added to the database by the test before it reaches it. Anyways, thanks for catching it, sent a PR to fix it.